src/Models/Constants/TelemetryConstants.ts

```diff
@@ -177,7 +179,81 @@ export const TelemetryEvents = {
         }
     },
     /** viewer section of the app */
-    Viewer: {}
+    Viewer: {},
+    /** tools provided by the library */
+    Tools: {
+        /** data history explorer component */
+        DataHistoryExplorer: {
+            SystemAction: {
+                ParseModal: {
```

Comment:
Suggestion for a name change: `ParseDataHistory`, Modal is not very representative of the data.

Reply:
make sense, changing it